src/components/Comments.js

```diff
@@ -96,7 +96,7 @@ const Comments = ({ article_id }) => {
           onClick={(event) => handleSubmit(event)}
           className="comment-submit"
```

Comment:
It might be really nice to enable multi-line comment bodies which would require you to change the <input > tag to a <text area> tag